.changeset/big-frogs-rest.md

```diff
@@ -0,0 +1,15 @@
+---
+"saleor-app-emails-and-messages": patch
+"saleor-app-data-importer": patch
+"saleor-app-products-feed": patch
+"saleor-app-monitoring": patch
+"saleor-app-invoices": patch
+"saleor-app-klaviyo": patch
+"saleor-app-search": patch
+"saleor-app-slack": patch
+"saleor-app-taxes": patch
+"saleor-app-cms": patch
+"saleor-app-crm": patch
+---
+
+Removed `typescript-urql` package. All packages use `TypedDocument` for making GraphQL operations.
```

Comment:
Please remember that changelog is not a commit. It will be read for single app, not "monorepo". So for user there is no "packages" here, its only a single app. And the changelog is duplicated.